package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
```

Comment:
Yes - I don't see that anything else makes sense - given we configure "devTag" as "dev" in our options, it seems clear that it is the task of the plugin to append it. We should just have release version numbers (of the next planned release) in committed package.json, I think